File: Python/01.ProgrammingBasics/06.Conditional-Statements-Adv-Exercise/07.HotelRoom/_07.HotelRoom.py
month = input()
nights = int(input())

apartment_price = 0
studio_price = 0

if month == "May" or month == "October":
    apartment_price = 65 * nights
    studio_price = 50 * nights
    if nights > 14:
        studio_price *= 0.70
        apartment_price *= 0.90
    elif nights > 7:
        studio_price *= 0.95
elif month == "June" or month == "September":
    apartment_price = 68.70 * nights
    studio_price = 75.20 * nights
    if nights > 14:
        studio_price *= 0.80
        apartment_price *= 0.90
elif month == "July" or month == "August":
    apartment_price = 77 * nights
    studio_price = 76 * nights
    if nights > 14:
        apartment_price *= 0.90

print(f"Apartment: {apartment_price:.2f} lv.")
print(f"Studio: {studio_price:.2f} lv.")

File: C++/01-Basics/03.3-ConditionalStatementsAdvЕxercise/07.HotelRoom/07.HotelRoom.cpp
#include <iostream>
#include <iomanip>

using namespace std;

int main() {
    string month;
    int nights;
    cin >> month >> nights;

    double studioPrice, apartmentPrice;

    if ((month == "May" || month == "October") && nights > 14) {
        studioPrice = 50 * 0.70;
    }
    else if ((month == "May" || month == "October") && nights <= 14 && nights > 7) {
        studioPrice = 50 * 0.95;
    }
    else if ((month == "May" || month == "October") && nights <= 7) {
        studioPrice = 50;
    }
    else if ((month == "June" || month == "September") && nights > 14) {
        studioPrice = 75.20 * 0.80;
    }
    else if ((month == "June" || month == "September") && nights <= 14) {
        studioPrice = 75.20;
    }
    else if (month == "July" || month == "August") {
        studioPrice = 76;
    }
    
    

    if (nights > 14) {
        if (month == "May" || month == "October") {
            apartmentPrice = 65 * 0.90;
        }
        else if (month == "June" || month == "September") {
            apartmentPrice = 68.70 * 0.90;
        }
        else if (month == "July" || month == "August") {
            apartmentPrice = 77 * 0.90;
        }
    }
    else {
        if (month == "May" || month == "October") {
            apartmentPrice = 65;
        }
        else if (month == "June" || month == "September") {
            apartmentPrice = 68.70;
        }
        else if (month == "July" || month == "August") {
            apartmentPrice = 77;
        }
    }

    double totalStudioPrice = studioPrice * nights;
    double totalApartmentPrice = apartmentPrice * nights;

    cout << "Apartment: " << fixed << setprecision(2) << totalApartmentPrice << " lv." << endl;
    cout << "Studio: " << fixed << setprecision(2) << totalStudioPrice << " lv." << endl;

    return 0;
}
